### PR TITLE
Automate case: start guest with volume backingStore disk

### PR DESCRIPTION
**Pull request by chunfuwen**
Automate case: start guest with disk backingStore type = volume [1]

Vm can be started sucessfully when Vm has additional disk with volume backingStore type

[1]https://bugzilla.redhat.com/show_bug.cgi?id=1804603

Signed-off-by: chunfuwen <chwen@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
Command 'virsh list' finished with 0 after 0.032903457s
cleaning libvirtd logs...
DATA (filename=output.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stdout.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stderr.expected) => NOT FOUND (data sources: variant, test, file)
PASS 1-type_specific.io-github-autotest-libvirt.virtual_disks.backingstore_disk.coldplug.volume_backingstore.start_vm

**Comment by chunfuwen** [comment]
Automate xx-189187

**Comment by chunfuwen** [comment]
@qiankehan  please take a look

**Comment by chunfuwen** [comment]
@qiankehan  please take a look

**Review by qiankehan — APPROVED** [review]
LGTM

**Review by dzhengfy — COMMENTED** [review]
Others LGTM

File: libvirt/tests/src/virtual_disks/virtual_disks_backingstore_disk.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+    hotplug = "yes" == params.get("virt_device_hotplug")
+    status_error = "yes" == params.get("status_error")
+
+    # Back up xml file
+    if vm.is_alive():
+        vm.destroy(gracefully=False)
+    vmxml_backup = vm_xml.VMXML.new_from_inactive_dumpxml(vm_name)
+    try:
+        vmxml = vm_xml.VMXML.new_from_dumpxml(vm_name)
```
[the remark this answers is not in the document]
Comment (chunfuwen) [reply libvirt/tests/src/virtual_disks/virtual_disks_backingstore_disk.py]: updated

**Review by dzhengfy — APPROVED** [review]

**Pull request merged**